python/rrc_example_package/cube_env.py

```diff
@@ -221,6 +217,11 @@ def step(self, action):
                 self.info,
             )
 
+            self.step_count = t
+            # make sure to not exceed the episode length
+            if self.step_count >= move_cube.episode_length - 1:
```

Comment:
why the -1?

also, not sure what's the expected behaviour, but it seems to me that people will still be able to execute single actions after exceeding the episode length, or am i missing something?

Reply:
-1 because the step count starts at 0.
The backend automatically shuts down when the episode is over, so sending more actions results in an error